apps/api/src/database/schema.prisma

```diff
@@ -14,7 +14,53 @@ datasource db {
 }
 
 model User {
-  id    Int     @id @default(autoincrement())
-  email String  @unique
-  name  String?
+  id        Int      @id @default(autoincrement())
+  name      String
+  email     String
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @updatedAt @map("updated_at")
+
+  subscriptions Subscription[]
+
+  @@map("users")
+}
+
+model Company {
+  id        Int      @id @default(autoincrement())
+  name      String
+  image     String
+  createdAt DateTime @default(now()) @map("created_at")
```

Comment:
Oof nice catch! This is because databases usually has a different naming convention (plural and snake_case) from JS naming convention (singular and camelCase) and we want both to coexist.

This is described in more details in this [Prisma doc](https://www.prisma.io/docs/orm/prisma-schema/data-model/database-mapping). Schema file gets a little bit more verbose but code and database is standardized in a nice way

In short, in the left you read as field names while coding (client generated by Prisma) and on the right (the @map field) is how you will see in the database tables